src/3box.js

```diff
@@ -66,82 +76,55 @@ class ThreeBox {
       if (peer === pinningNode.split('/').pop()) {
         console.timeEnd('opening pinning room, pinning node joined')
         console.log('broadcasting odb-address')
-        this._pubsub.publish(PINNING_ROOM, { odbAddress: rootStoreAddress })
+        this._pubsub.publish(PINNING_ROOM, {type: 'PIN_DB', odbAddress: rootStoreAddress })
       }
     }
-    if (rootStoreAddress) this._pubsub.subscribe(PINNING_ROOM, () => {}, onNewPeer)
 
-    const keystore = new OrbitdbKeyAdapter(this._muportDID)
-    console.time('new OrbitDB')
-    this._orbitdb = new OrbitDB(this._ipfs, opts.orbitPath, { keystore })
-    console.timeEnd('new OrbitDB')
-    globalIPFS = this._ipfs
-    globalOrbitDB = this._orbitdb
+    const parsePubSubMessage = (resolve) => (topic, data) => {
+      if (data.type === 'HAS_ENTRIES' & data.odbAddress === rootStoreAddress) {
```

Comment:
I'd prefer to use a logical and (&&) instead of a bitwise and.

Reply:
yup definitely